code/orders-api-rest/api/openapi-rest.yml

```
@@ -807,122 +801,179 @@ paths:
       tags:
-        - UserPersonalCabinet
-      summary: Update personal info
-      description: Update personal information such as first name, last name, and phone number for the authenticated user.
-      operationId: updatePersonalUserInfo
-      security:
-        - bearerAuth: [ ]
+        - PasswordResetController
+      summary: Reset password with one-time token
+      description: |
+        Accepts a valid one-time token and new password to reset the user's password.
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)